vendor/src/github.com/docker/engine-api/types/registry/registry.go

```diff
@@ -82,7 +82,7 @@ type SearchResult struct {
 	IsOfficial bool `json:"is_official"`
 	// Name is the name of the repository
 	Name string `json:"name"`
-	// IsOfficial indicates whether the result is trusted
+	// IsTrusted indicates whether the result is trusted
```

Comment:
Actually, I'm wrong: https://docs.docker.com/engine/reference/api/docker_remote_api_v1.19/.

"This API returns both is_trusted and is_automated images. Currently, they are considered identical. In the future, the is_trusted property will be deprecated and replaced by the is_automated property."

Seems to have been deprecated between 1.19 and 1.20. I propose we remove this from engine-api.